Doc/howto/urllib2.rst

```diff
@@ -15,7 +15,7 @@ Introduction
     You may also find useful the following article on fetching web resources
     with Python:
 
-    * `Basic Authentication <https://web.archive.org/web/20201215133350/http://www.voidspace.org.uk/python/articles/authentication.shtml>`_
+    * `Basic Authentication <https://web.archive.org/web/20201215133350/http://www.voidspace.org.uk/python/articles/authentication.shtml>`__
```

Comment:
I guess we should generally use anonymous targets as a default, unless we specifically want a named one?

Needing a named one is probably rare.

If so, we could update the [examples in the devguide](https://devguide.python.org/documentation/markup/), perhaps with a brief explanation.

And suggest anonymous in reviews.

Reply:
PR opened: https://github.com/python/devguide/pull/1709